jobs/check-aws-regions.Jenkinsfile

```
@@ -26,7 +26,7 @@ cosaPod(serviceAccount: "jenkins"){
     }
     if (disabled_regions != "None") {
         warn("Disabled AWS regions detected: ${disabled_regions}")
-        pipeutils.trySlackSend(message: ":aws: aws-regions-disabled :jenkins:<${env.BUILD_URL}|${env.BUILD_NUMBER}> ${disabled_regions}")
+        pipeutils.trySlackSend(message: ":aws: aws-regions-disabled :jenkins:<${env.BUILD_URL}|${env.BUILD_NUMBER}> :ocean:<${env.RUN_DISPLAY_URL}|${env.BUILD_NUMBER}> ${disabled_regions}")
```

Thanks for this. I've been thinking it would be nice to make the emoji itself clickable so we only have to include the build # once (we would need to apply this to all jobs and not just this new one).

Also I think it would be better to have the first bit of output from this job be the name of the job `check-aws-regions`. 

```suggestion
        pipeutils.trySlackSend(message: ":aws: check-aws-regions #${env.BUILD_NUMBER} <${env.BUILD_URL}|:jenkins:> <${env.RUN_DISPLAY_URL}|:ocean:> detected disabled regions: ${disabled_regions}")
```

---

sure,  will make this change to all the jobs